server.c

```diff
@@ -201,7 +206,18 @@ int run_server(const char *port, const char *cert, const char *key)
         return 1;
     }
 
-    printf("starting server on port %s\n", port);
+    uint64_t pid;
+
+    #ifdef __APPLE__
+        pthread_threadid_np(NULL, &pid);
+    #else
+        pid = syscall(SYS_gettid);
+    #endif
+
+    char pid_char[21];
+    sprintf(pid_char, "%" PRIu64, pid);
+
+    printf("starting server with pid %s on port %s\n", pid_char, port);
```

Comment:
The `pid_char` and `sprintf` can be combined into the `printf` 
```printf("starting server with pid %" PRIu64 " on port %s\n", pid, port);```

Reply:
My bad - Fixed in https://github.com/rbruenig/qperf/pull/5/commits/9e98777372bac53743104be14522d53ce0e35de4